RC Reorg - Accounts and settings (#2034)
RedisLabs/redislabs-docs: content/rc/cloud-integrations/aws-cloud-accounts/_index.md

To do so, you associate your existing AWS account as a _cloud account_ for your subscription.  This requires setting up and entering credentials that enable monitoring, maintenance, and technical support of your subscription.

To do this, you need:

1. A programmatic user with an access key and a secret access key for that user.
1. A console role that allows administrative access to the cloud account.

These resources need to exist before adding the cloud account to your subscription.  To learn more, see [Create IAM resources]({{<relref "/rc/cloud-integrations/aws-cloud-accounts/iam-resources">}}).

{{<note>}}
Once an AWS account has been configured as a cloud account, you must _not_:
- Manually change the configuration of required resources, such as security groups<br/>
- Manually suspend or stop (terminate) provisioned resources</br>
{{</note>}}

## View cloud account settings

To create or edit a cloud account in Redis Enterprise Cloud:

> 1. Sign into the [admin console](https://app.redislabs.com/) and then select the target subscription.

Reviewer: ```suggestion
1. Sign in to the [admin console](https://app.redislabs.com/) and then select the target subscription.
```

Author: Fixed.

{{<image filename="images/rc/icon-cloud-account-add.png" alt="Use the Add button to add new cloud accounts to your Redis Cloud subscription." width="36px">}}{{< /image >}}

This displays the **Add cloud account** dialog

{{<image filename="images/rc/account-settings-prompt-add-cloud-account.png" alt="Use the Add cloud account prompt to enter the details of the cloud account." width="50%">}}{{< /image >}}

Each of the following fields are required.

|Setting|Description|
|-------|-----------|
| _Account name_ | A descriptive name for your cloud account settings |
| _AWS&nbsp;access&nbsp;key_ | The AWS access key for the programmatic user created to support your cloud account settings |
| _AWS&nbsp;secret&nbsp;key_ | The AWS secret key for the programmatic user created to support your cloud account settings |
| _IAM role name_ | The name of the AWS console role with access to the AWS console |

Use the **Add account** button to save your cloud account details.

{{<image filename="images/rc/button-cloud-account-add.png" alt="Use the Add account button to save the details of your new cloud account." width="140px">}}{{< /image >}}

> Be sure to create the resources before adding the cloud account to your subscription, as they're used to verify access to the cloud account.  The details can be saved oly after access is verified.

Reviewer: ```suggestion
Be sure to create the resources before adding the cloud account to your subscription, as they're used to verify access to the cloud account.  The details can be saved only after access is verified.
```

Author: Fixed.

When problems occur, an information icon appears and the field is highlighted in red.  When this happens, the icon includes a tooltip that explains the issue.

{{<image filename="images/rc/account-settings-prompt-cloud-account-error.png" alt="When errors occur, the field is highlighted in red and a notification icon appears.  The icon tooltip describes the problem." width="50%">}}{{< /image >}}

If the **Add account** button is inactive, verify that:

- You've specified all field values correctly
- The resources exist in your AWS account
- Each resource provides the required level of access

For help, see [Create IAM resources]({{<relref "/rc/cloud-integrations/aws-cloud-accounts/iam-resources">}}).

## Edit cloud account details

To update the details of a cloud account associated with your Redis Cloud subscription, select the cloud account from the **Cloud account** tab and then select the **Edit** button.

{{<image filename="images/rc/icon-cloud-account-edit.png" alt="Use the Edit button to update cloud account details." width="36px">}}{{< /image >}}

> This displays the **Edit cloud account dialog**:

Reviewer: ```suggestion
This displays the **Edit cloud account** dialog:
```

Author: Fixed.

{{<image filename="images/rc/account-settings-prompt-edit-cloud-account.png" alt="Use the Edit cloud account prompt to update the details of the cloud account." width="50%">}}{{< /image >}}

|Setting|Description|
|-------|-----------|
| _Account name_ | A descriptive name for your cloud account settings |
| _AWS access key_ | The AWS access key for the programmatic user created to support your cloud account settings |
| _AWS secret key_ | The AWS secret key for the programmatic user created to support your cloud account settings |
| <nobr>_AWS console username_</nobr> | The username for the AWS console |
| _AWS console password_ | The password for AWS console access |

> Use the **Update account** button so save your changes.

Reviewer: ```suggestion
Use the **Update account** button to save your changes.
```

Author: Fixed.

{{<image filename="images/rc/button-cloud-account-update.png" alt="Use the Update account button to save the updated cloud account details." width="140px">}}{{< /image >}}

## Delete cloud account details

To remove a cloud account from your Redis cloud subscription, select the cloud account from the **Cloud account** tab and then select the **Delete** button.

{{<image filename="images/rc/icon-cloud-account-delete.png" alt="Use the Delete button to remove cloud account details." width="36px">}}{{< /image >}}

## Dedicated IAM resources

> We strongly recommend using dedicated identity and access management (IAM) resources to manage your ASW cloud accounts.  These resources should not be shared with any other task, account, or process.

Reviewer: ```suggestion
We strongly recommend using dedicated identity and access management (IAM) resources to manage your AWS cloud accounts.  These resources should not be shared with any other task, account, or process.
```
